bancor_research/bancor_simulator/v3/spec/network.py

```diff
@@ -52,6 +52,7 @@ def __init__(
         whitelisted_tokens=DEFAULT.WHITELIST,
         price_feeds_path: str = DEFAULT.PRICE_FEEDS_PATH,
         price_feeds: PandasDataFrame = DEFAULT.PRICE_FEEDS,
+        log_state: bool = False,
```

Comment:
Need to add this also in `/bancor_emulator/v3/spec/network.py` (even if not used there at the moment), so that if we have a place where we state this flag explicitly upon the creation of a `BancorDapp` instance, then we can easily duplicate the instantiation for both the simulator's class and the emulator's class.